test/typescript-validate.ts

```diff
@@ -149,3 +151,5 @@ export function webhookErrorTestDeprecated(error: WebhookError) {
   console.log(event);
   console.log(request);
 }
+
+sign("randomSecret", {});
```

Comment:
Should we move this to deprecated part of typescript-validate? or not yet?

Reply:
no I wouldn't consider it deprecated, both passing a string or an object are valid, I don't expect we will deprecate either anytime soon